server/apps/publish/archive_publish.py

```diff
@@ -718,3 +718,9 @@ def on_update(self, updates, original):
     include_states=['published', 'corrected'],
     privileges=['correct']
 )
+
+superdesk.workflow_action(
```

Comment:
Right, it's not yet merged. It's in [PR#867](https://github.com/superdesk/superdesk/pull/867)